cmd/cli/serve/util.go

```diff
@@ -167,6 +168,27 @@ func SetupIPFSClient(ctx context.Context, cm *system.CleanupManager, ipfsCfg typ
 	return client, nil
 }
 
+func StartWebUIServer() (error) {
+     // Define command to start the React server
```

Comment:
I don't think this should be the way we launch our web ui as it's adding a big dependency on deployments that they need to have node/npm installed and ideally we wouldn't do that.  

We _should_ have a build step that uses NPM to build a deployable standalone version of the content/JS/CSS.   Then we can embed that into this binary (something like https://harsimranmaan.medium.com/embedding-static-files-in-a-go-binary-using-go-embed-bac505f3cb9a) and then implement a webserver in the app to serve that embedded content.